module2/lessons/rails_resources.md

```diff
@@ -91,47 +99,47 @@ end
 
 Just like before, we only want to create the routes we need.
 
-With a partner, refactor the nested routes in SetList to use the `resources` sytnax.
+With a partner, refactor the nested routes in SetList to use the `resources` sytnax.
 
 ## Route Helpers
 
-If you run `rake routes`, you'll notice the first column is called "prefix". Rails will use the "prefix" column to build route helpers.
+If you run `rake routes`, you'll notice the first column is called "prefix". Rails will use the "prefix" column to build route helpers.
 
-Route helpers will generate a path for you (note: just the path, not the VERB). All you have to do is append `_path` to the end of the prefix name. For example, if you have this in your routes:
+Route helpers will generate a path for you (note: just the path, not the VERB). All you have to do is append `_path` to the end of the prefix name. For example, if you have this in your routes:
 
 ```ruby
-resources :cats, only: [:index]
+resources :dogs, only: [:index]
 ```
 
-Then `rake routes -c cats` should give you:
+Then `rake routes -c dogs` should give you:
 
-```
+```bash
 Prefix Verb URI Pattern     Controller#Action
-  cats GET  /cats(.:format) cats#index
+  dogs GET  /dogs(.:format) dogs#index
 ```
 
-Using that prefix `cats`, we can use `cats_path` anywhere in our Rails app to generate the path `/cats`.
+Using that prefix `dogs` we can use `dogs_path` anywhere in our Rails app to generate the path `/dogs`.
 
-Generally, any row in your `rake routes` output that does not include a prefix uses the same prefix as the line above it.
+Generally, any row in your `rails routes` output that does not include a prefix uses the same prefix as the line above it.
 
 ## Passing Parameters to Route Helpers
 
 Some paths include parameters. For example:
 
 ```ruby
-resources :cats, only: [:show]
+resources :dogs, only: [:show]
 ```
 
-Gives you this `rake routes` output:
+Gives you this `rails routes` output:
 
-```
+```bash
 Prefix Verb URI Pattern         Controller#Action
-   cat GET  /cats/:id(.:format) cats#show
+   dog GET  /dogs/:id(.:format) dogs#show
 ```
 
-You can't generate the path using `cat_path` because it is expecting an `:id`. Any time a route helper needs a dynamic parameter, like `:id` we MUST pass a value to the route helper. For example `cat_path(29)` will generate `/cats/29`.
+You can’t generate the path using `dog_path`  because it is expecting to be passed an `:id`. Any time a route helper needs a dynamic parameter like `:id`, we MUST pass a value to the route helper. For example, `dog_path(29)` will generate `/cats/29`.
```

Comment:
```suggestion
You can’t generate the path using `dog_path`  because it is expecting to be passed an `:id`. Any time a route helper needs a dynamic parameter like `:id`, we MUST pass a value to the route helper. For example, `dog_path(29)` will generate `/dogs/29`.
```